Various/rodilab_Color palette.lua

```diff
@@ -1,14 +1,9 @@
 -- @description Color palette
 -- @author Rodilab
--- @version 2.0
+-- @version 2.10
```

Comment:
10 would prevent releasing v2 updates with minor version numbers 1 through 9. Is this OK?